core/src/main/scala/org/apache/spark/errors/SparkCoreErrors.scala

```diff
@@ -325,4 +325,24 @@ object SparkCoreErrors {
     new SparkException(errorClass = "GRAPHITE_SINK_PROPERTY_MISSING",
       messageParameters = Array(missingProperty), cause = null)
   }
+
+  def storeBlockError(blockId: BlockId, cause: Throwable = null): Throwable = {
+    new SparkException(errorClass = "STORE_BLOCK_ERROR",
+      messageParameters = Array(blockId.toString), cause = cause)
+  }
+
```

Comment:
Let us remove `getBlockError`, `corruptedRemoteBlockError` and `getLocalBlockError` - they are replacing a single invocation site.

Reply:
Not sure what you mean by "replacing a single invocation site". Could you elaborate?